astroquery/mast/core.py

```diff
@@ -1730,18 +1741,22 @@ def progress_callback(numbytes):
             bkt.download_file(bucket_path, local_path, ExtraArgs={"RequestPayer": "requester"},
                               Callback=progress_callback)
 
-    def _download_files(self, products, base_dir, cache=True):
+    def _download_files(self, products, base_dir, cache=True, cloud_only=False,):
         """
-        Takes an `astropy.table.Table` of data products and downloads them into the dirctor given by base_dir.
+        Takes an `~astropy.table.Table` of data products and downloads them into the dirctory given by base_dir.
 
         Parameters
         ----------
-        products : `astropy.table.Table`
+        products : `~astropy.table.Table`
             Table containing products to be downloaded.
         base_dir : str
             Directory in which files will be downloaded.
         cache : bool
-            Default is True. If file is found on disc it will not be downloaded again.
+            Default is True. If file is found on disk it will not be downloaded again.
+        cloud_only : bool, optional
+            Default False. If set to True and cloud data access is enabled (see `enable_cloud_dataset`)
```

Comment:
add double quotes, don't think otherwise sphinx will be happy

Reply:
Where are you suggesting I add double quotes?

Reply:
never mind me, these are available in the API docs as links (as they are defined in the same namespace). Not sure where my mind was when I did the review 😅